### DISCUSSION TITLE
Choosing the location of replicas in cstor

### DISCUSSION THREAD

**Discussion by mohammada315** [openebs/openebs#3692] [Q&A]
dear openebs developers
How to specify, for example, a PVC to create a replica in specified nodes

**Comment by orville-wright** [comment]
Hi @mohammada315 
Please note: the cStor project has been archived and is no longer a core active component of openEBS.
- Please see ``` Project announcement ``` here: https://github.com/openebs/openebs/issues/3709

- the repo and all dependencies have been migrated to the CNCF owned [openEBS Archive](https://github.com/openebs-archive)
- Support for cStor will be done on a ```best effort``` basis cvia the CNCF openEBS Slack channel

We recommend users migrate their cStor deployments to other OpenEBS Storage Engines.
- If you need Replicated PV Storage, we recommend you look at: [```openEBS Maystor```](https://github.com/openebs/mayastor)
